src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataEBDRequest.java

```diff
@@ -0,0 +1,234 @@
+package com.hms_networks.americas.sc.extensions.historicaldata;
+
+import com.hms_networks.americas.sc.extensions.system.time.SCTimeUnit;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Utility methods for building EBD requests for historical data according to Export Block
+ * Descriptor REFERENCE GUIDE RG-0009-00-EN, Publication date 30/08/2023
+ */
+public class HistoricalDataEBDRequest {
```

Comment:
Class name violates Google Code Format. It should be `HistoricalDataEbdRequest`. This is also consistent with other instances of 'EBD' in class or variable names in the project.

Google Java Format Conventions for Naming: https://google.github.io/styleguide/javaguide.html#s5-naming

Reply:
changed